jsonschema2pojo-core/src/main/java/org/jsonschema2pojo/Schema.java

```diff
@@ -59,12 +59,20 @@ public JsonNode getContent() {
         return content;
     }
 
-    public JsonNode getParentContent() {
-        return parentContent;
+    public Schema getParent() {
+        return parent;
     }
     
     public boolean isGenerated() {
         return javaType != null;
     }
 
+    public Schema derive(JsonNode content) {
```

Comment:
Could you add some javadoc to this method? I don't think it is self-explanatory based on the name alone.